doc/1-intro/glossary.md

```diff
@@ -0,0 +1,145 @@
+- usually stored in .BRR files.
+
+**clipping**: when a sample or playback stream exceeds the maximum or minimum values. this can cause audible distortion.
+- this often occurs when a sample is amplified too much.
+- it can also occur during playback if too much sound is being added together at once. in some cases the mixer can be used to reduce the volume. if this doesn't work, the clipping is caused within the chip's own mixing, and the only solution is to reduce the volumes of the notes being played.
+
+**clock rate**: the timing at which a chip operates, expressed as cycles per second (Hz).
+- changing this may change aspects of how some chips work, most notably pitch.
+- some chips cannot operate at anything other than their designed clock rate.
+
+**cursor**: the marker of input focus. anything typed will happen at the cursor's location.
+- _Furnace:_ this always refers to the pattern view except when in a text entry box.
+
+**DAC**: digital analog converter. this converts a digital representation of sound into actual output.
+
+**.DMF**: a DefleMask song file.
+- _Furnace:_ DFM files may be read, and compatibility flags will be set to make them play as accurately as possible, but there may still be glitches.
+- _Furnace:_ DFM files may be saved, but full compatibility isn't guaranteed and many features will be missing. this isn't recommended unless absolutely necessary.
+
+**.DMP**: a DefleMask instrument file.
+
+**.DMW**: a DefleMask wavetable file.
+
+**DPCM**: differential/delta pulse code modulation. this is a variety of PCM that stores each amplitude as its difference from the previous.
+
+**duty**: in a pulse wave, this represents the ratio of the "on" part of the wave.
+- a square wave is a pulse wave with a duty of 50%.
+
+**feedback**: in FM instruments, this adds some of an operator's output into itself to create complex harmonics.
+- in the algorithm view, an operator with a circle around it is capable of feedback.
+
+**FM**: frequency modulation. this is a method of generating sound that uses one operator's amplitude to modify another operator's frequency.
+- the FM in Yamaha chips is more accurately called _phase modulation,_ which uses a different method of computation to achieve similar results.
+
+**.FUI**: a Furnace instrument file.
+
+**.FUR**: a Furnace tracker file.
+
+**.FUW**: a Furnace wavetable file.
+
+**hard-pan**: sounds can only be panned all the way to one side or the other, not in-between.
+
+**Hz**: hertz (pronounced as "hurts"). a unit representing divisions of one second. 1 Hz means once per second; 100 Hz means one hundred times per second. also, _kHz_ (kilohertz, one thousand per second) and _MHz_ (megahertz, one million per second).
+
+**interpolate** (pattern): to fill in the area between two values with a smooth ramp of values in between.
+
+**interpolation** (sample): filtering of sample output to remove unintended harmonics and achieve a smoother sound.
+- analogous to image antialiasing.
```

Comment:
Wrong.

Reply:
You'll have to explain why.

Reply:
Actually, nevermind, I'll just pull it.

Reply:
Interpolation **involves** filling in the missing area between two samples.